config/routes/user.rb

```diff
@@ -0,0 +1,13 @@
+devise_for :users, controllers: { registrations:      "auth/registrations",
```

Comment:
Rename file to `users` instead of `user`?

Reply:
done.